crypto/keys/keybase.go

```diff
@@ -179,6 +180,13 @@ func (kb dbKeybase) List() ([]Info, error) {
 	iter := kb.db.Iterator(nil, nil)
 	defer iter.Close()
 	for ; iter.Valid(); iter.Next() {
+		key := string(iter.Key())
+
+		// need to exclude any keys in storage that have an address suffix
```

Comment:
I think a more defensive mechanism would be to: `if strings.HasSuffix(key, infoSuffix) { // read and parse...}`